docs/source/gbd2017_models/risks/risk_exposure_template.rst

```diff
@@ -1,53 +1,180 @@
+      :code:`.. _2017_cause_model_template:` at the top of this file with a
+      unique reference label for your cause. The reference label should have the
+      form :samp:`.. _2017_exposure_{\{exposure_name\}}:`, where
+      :samp:`{\{exposure_name\}}` is replaced with a unique descriptive name or
+      abbreviation for your risk exposure, e.g. :code:`.. _2017_exposure_unsafe_water:`.
+
+   #. Delete this document's title above:
+
+      .. code:: reStructuredText
+
+         ----------------------
+         Risk Exposure Template
+         ----------------------
+
+      Once the above title is deleted, all the other section titles will be
+      promoted up one level.
+
+   #. The subtitle below should now be the document's title. Replace the text
+      in the below (sub)title with the full name of your exposure in GBD 2017. For example, if you were modeling the exposure "Low Birth Weight and Short Gestation," then the title
+
+      .. code:: reStructuredText
+
+         ======================================
+         Full Name of Risk Exposure in GBD 2017
+         ======================================
+
+      should be replaced by
+
+      .. code:: reStructuredText
 
-.. todo::
-  Add instructions, following Nathaniel's cause model template instructions. Flesh out all headings.
+         ====================================
+         Low Birth Weight and Short Gestation
+         ====================================
+
+      **Note:** Be sure to adjust the length of the title's underline
+      :code:`======` and overline :code:`======` to match the length of your
+      new document title, or you will get errors in the `section structure
+      <https://docutils.sourceforge.io/docs/user/rst/quickref.html#section-structure>`_
+      when Sphinx builds HTML from the :file:`index.rst` file.
+
+   #. Once you complete these steps, delete this :code:`.. important::`
+      `directive <https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#directives>`_
+      from :file:`index.rst`.
+
+
+======================================
+Full Name of Risk Exposure in GBD 2017
+======================================
 
 
 Risk Exposure Overview
 ----------------------
 
- - expert opinion
+Include here a clinical background and overview of the exposure you're modeling.
+Note that this is only for the exposure; you will include information on the 
+relative risk of the relevant outcomes, and the cause models for those outcomes, 
+in a different document.
 
-Risk Exposure description
--------------------------
 
 Risk Exposures Description in GBD
-------------------------------
+---------------------------------
+
+Include a description of this exposure model in the context of GBD, involving
+but not limited to:
 
-  - Type of risk (categorical, continuous, etc.)
+  - What type of statistical model? (categorical, continuous?)
 
-  - What is this a risk for?
+  - How is the exposure estimated? (DisMod, STGPR?)

```

Comment:
I think adding this section is very informative! I'm interested to see how 'What is this a risk for?' can be either a short answer or visualized using a hierarchy (if the risk affects many causes and their subcauses). Should there be a section to consider about 'risk-risk correlation', potentially? Or is that just based on a specific risk-cause pair?